src/comm.cc

```diff
@@ -592,7 +592,7 @@ commUnsetFdTimeout(int fd)
 }
 
 int
-commSetConnTimeout(const Comm::ConnectionPointer &conn, int timeout, AsyncCall::Pointer &callback)
+commSetConnTimeout(const Comm::ConnectionPointer &conn, time_t timeout, AsyncCall::Pointer &callback)
```

Comment:
I did not realize this by looking at the PR diff, but I now see two more problems related to this (merged) PR scope:

* This function and commUnsetConnTimeout() return type should be `time_t` rather than `int`.
* The C-style `timeout` cast inside this function should be removed.

@kinkie, if you can fix these problems, please do.

The latest Coverity report (defect 1547031) triggered this comment.